### PR TITLE
october edits

### PR DESCRIPTION
**Pull request by priyasilverstein**
resources additions
fixing steering page

### CODE REVIEW

File: docs/resources.html [hunk 1]
```diff
@@ -63,7 +64,8 @@ <h2 id="diversifying-social-science-research">Diversifying social science resear
...
 				<p>One step towards diversifying social science research is to have a better picture of who constitutes a journal’s authors, reviewers, and editors. Collecting demographic data may help efforts to increase diversity.</p>
-				<p>However, it is important to acknowledge that such collections may not be straightforward. For example, there may be geographical differences in how (and whether) to require authors to specify the demographic characteristics of their samples or themselves. For example, see <a href="https://doi.org/10.1002/icd.2260" target="_blank">Jugert et al. (2021)</a> and <a href="https://doi.org/10.1080/15283488.2021.1932901" target="_blank">Juang et al. (2021)</a> for discussions of these tensions within a European context.</p>
+				<p>Editors at <a href="https://onlinelibrary.wiley.com/journal/14756811" target="_blank">Personal Relationships</a> have created a “<a href="https://docs.google.com/document/d/1mIzvUz97ThJVwuVZOqE8Ab3Mw4hD1_zA/edit" target="_blank">iversity matrix</a>d” template that can be used and adapted to collect data from authors, reviewers, editorial board members, editors, etc.</p>
```
Comment (adam3smith) [inline docs/resources.html:67]: Typo here?

File: docs/resources.html [hunk 2]
```diff
@@ -80,10 +82,14 @@ <h2 id="open-science">Open science</h2>
...
 				<h3 id="data-and-code">Data and code</h3>
-				<p> A set of stable core practices has begun to emerge with regard to data management and sharing. While all are readily available, some require more effort on the part of journals. For example, requiring authors to share data via a trusted digital repository (and not e.g. via a personal website) is easier than checking that the author's code runs. </p>
+				<p> A set of stable core practices has begun to emerge with regard to data management and sharing. While all are readily available, some require more effort on the part of journals. For example, requiring authors to share data via a trusted digital repository (and not e.g. via a personal website) is easier than checking that the author's code runs. Iain Hrynaszkiewicz has written a chapter outlining “<a href="https://doi.org/10.1007/164_2019_290" target="_blank">Publishers’ Responsibilities in Promoting Data Quality and Reproducibility</a>” that describes practical approaches being taken by publishers to promote rigor and transparency in data practices.</p>
```
Comment (adam3smith) [inline docs/resources.html:85]: Is Iain's affiliation relevant here? I'd be inclined to include it in parentheses

File: docs/resources.html [hunk 3]
```diff
@@ -80,10 +82,14 @@ <h2 id="open-science">Open science</h2>
...
+				<p><a href="https://www.force11.org/" target="_blank">FORCE11</a> and <a href="https://publicationethics.org/" target="_blank">COPE</a> have developed some recommendations for the handling of ethical concerns relating to the publication of research data. See their <a href="https://www.force11.org/article/recommendations-handling-ethical-concerns-relating-publication-research-data" target="_blank">blog post</a> and the <a href="https://zenodo.org/record/5391293#.YYATD9bP0-Q" target="_blank">recommendations</a> themselves here.</p>
+				<p>There may be particular types of data that are more difficult to share -- e.g. data on sensitive topics. For a case study that includes challenges, tools, and future directions of sharing data in these cases, please see <a href="https://doi.org/10.1177%2F2515245917744281" target="_blank">Joel et al. (2018)</a>. For a case study that discusses the redaction of sensitive data, see <a href="https://doi.org/10.1128/mBio.00991-13" target="_blank">Casadevall et al. (2013)</a>.</p>
```
Comment (adam3smith) [inline docs/resources.html:92]: Comma after e.g. for consistency?

File: docs/resources.html [hunk 4]
```diff
@@ -139,9 +144,10 @@ <h3 id="open-peer-review">Open Peer Review</h3>
...
+				<p>In <b>open reviewing</b> everyone knows the identity of everyone. This is argued to increase the accountability of the reviewer, giving less scope for biased or unjustified judgements. <a href="https://doi.org/10.1001/jama.287.21.2762" target="_blank">TEXT</a>Godlee et al. (2002) offers a good introduction into the benefits of making reviewers open. </p>
```
Comment (adam3smith) [inline docs/resources.html:147]: I assume TEXT is a placeholder?

**Review by adam3smith — CHANGES_REQUESTED** [review]
A couple of typos and a couple if questions.

As a more general note, there's a lot of passive voice in this page. I understand why, but at least to me it reads a little weird at times. Might be worth talking about ideas to rephrase in general

**Comment by priyasilverstein** [comment]
thanks for catching those! have fixed. will leave passive voice for now, and think about editing in the next round. thanks!

**Review by adam3smith — CHANGES_REQUESTED** [review]
Sorry, found one more typo. (I usually just fix those, but currently most code review I do is on the phone, and the app doesn't allow that)

File: docs/governance.html [hunk 5]
```diff
@@ -47,6 +47,8 @@ <h3 id="steering-committee">Steering committee (alphabetized by first name)</h3>
 <li>Margaret Levenstein, Interuniversity Consortium for Political and Social Research, University of Michigan</li>
 <li>Mercè Crosas, Dataverse Project, Harvard University</li>
 <li>Moin Syed, Infant and Child Development, University of Minnesota</li>
+<li>Rick Gilmore, Databry, Pennsylvania State University</li>
```
[the remark this answers is not in the document]
Comment (priyasilverstein) [reply docs/governance.html]: fixed! thanks

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**